web/api/evaluations.py

```diff
@@ -0,0 +1,82 @@
+"""
+Evaluation API routes
+"""
+import json
+import random
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+from .. import crud, schemas, models
+from ..common import get_db, get_token
+
+router = APIRouter()
+
+
+@router.get("/{evaluation_id}", response_model=schemas.EvaluationDetail)
+def get_evaluation(
+    evaluation_id: int,
```

Comment:
do we really want to use integer ids in the API? It would be kinda nice to allow something more meaningful, a git shortref... but I guess this is OK.